packages/wonder-blocks-form/src/components/__tests__/labeled-text-field.test.js

```diff
@@ -0,0 +1,50 @@
+//@flow
+import * as React from "react";
+import {mount} from "enzyme";
+
+import LabeledTextField from "../labeled-text-field.js";
+
+const wait = (delay: number = 0) =>
+    new Promise((resolve, reject) => {
+        // eslint-disable-next-line no-restricted-syntax
+        return setTimeout(resolve, delay);
+    });
```

Comment:
No change required.

I think we should copy over my new `wait` implementation from webapp into Wonder Blocks and then reuse it across our tests rather than defining this in every test.

https://github.com/Khan/webapp/blob/master/services/static/javascript/testing/wait.js

Reply:
I could possibly partake in this (if the team's okay with this new change). Under what directory would this `"wait.js"` file go within Wonder Blocks?